tests/test_admin_server.py

```diff
@@ -23,6 +23,23 @@ def test_admin_endpoints(dbos: DBOS) -> None:
     assert response.status_code == 200
     assert response.json() == []
 
+    # Test GET /dbos-perf
+    response = requests.get("http://localhost:3001/dbos-perf", timeout=5)
+    assert response.status_code == 200
+    perf_util = response.json()
+    assert perf_util is not None
+    assert "idle" in perf_util
+    assert "active" in perf_util
+    assert "utilization" in perf_util
+
+    # Test GET /dbos-perf again, should be non-zero utilization
+    time.sleep(0.2)
+    response = requests.get("http://localhost:3001/dbos-perf", timeout=5)
+    assert response.status_code == 200
+    perf_util = response.json()
+    assert perf_util["utilization"] > 0.0
```

Comment:
Why can't it be 0.0?

Reply:
Because I want to make sure the utils are actually valid.

Reply:
Anyways the Github action env is weird.